saleor/graphql/tax/mutations/tax_configuration_update.py

```diff
@@ -155,36 +156,42 @@ def clean_input(cls, info: ResolveInfo, instance, data, **kwargs):
     @classmethod
     def clean_tax_app_id(cls, info: ResolveInfo, data):
         identifiers = []
-        if app_identifier := data.get("tax_app_id"):
+        if (app_identifier := data.get("tax_app_id")) is not None:
             identifiers.append(app_identifier)
 
         update_countries_configuration = data.get("update_countries_configuration", [])
         for country in update_countries_configuration:
-            if app_identifier := country.get("tax_app_id"):
+            if (app_identifier := country.get("tax_app_id")) is not None:
                 identifiers.append(app_identifier)
 
         active_tax_apps = list(get_active_tax_apps(identifiers))
+        active_tax_app_identifiers = [app.identifier for app in active_tax_apps]
 
-        if app_identifier := data.get("tax_app_id"):
-            cls.__verify_tax_app_id(info, app_identifier, active_tax_apps)
+        # include plugin in list of possible tax apps
+        active_tax_app_identifiers.append(AvataxPlugin.PLUGIN_IDENTIFIER)
```

Comment:
This should be fetched dynamically from PluginsManager. We can't hardcode it. 
Also the identifier for plugins should be returned with specific prefix, applied by Saleor. 
You could extend the manager in a similar way as it is done in `saleor.plugins.manager.PluginsManager.is_event_active_for_any_plugin`